documentation/architecture_guide.asciidoc

```diff
@@ -141,7 +141,7 @@ The provided _Data Layer_ contains the abstract classes to inherit from. All new
 
 [[img-t-architecture]]
 .Architecture representation in deep
-image::images/laryer_arch_detail.png["technical architecture", width="750", link="images/laryer_arch_detail.png"]
+image::images/layer_arch_detail.png["technical architecture", width="750", link="images/layer_arch_detail.png"]
```

Comment:
is this link working now? is image getting loaded?

Reply:
I have updated new image with new name. Now it's working.